OracleGoldenGate/21c/README.md

```diff
@@ -55,9 +55,32 @@ Sending build context to Docker daemon
 Successfully tagged oracle/goldengate-for-bigdata:21.4.0.0.0   
 ```
 
+To create a container image for GoldenGate for MySQL Database, use the following script:
+
+```sh
+$ docker build --tag=oracle/goldengate-for-MySQL:21.3.0.0.0 \
+                --build-arg INSTALLER=213000_ggs_Linux_x64_MySQL_services_shiphome.zip .
+Sending build context to Docker daemon
+...
+Successfully tagged oracle/goldengate-for-MySQL:21.3.0.0.0
+```
+
+To create a container image for GoldenGate for PostgreSQL Database, use the following script:
+
+```sh
+$ docker build --tag=oracle/goldengate-for-PostgreSQL:21.3.0.0.0 \
```

Comment:
Instead of all these examples, it would be cleaner to have *one* example with the `zip` file name as a parameter.